### PR TITLE
Post collector

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

**Comment by carusogabriel** [comment]
@TomasVotruba Gonna review it in my launch :smile:

**Comment by TomasVotruba** [comment]
Bonne appetit :D feel free to merge if all is fine

File: source/_posts/2018/2018-03-08-why-is-collector-pattern-so-awesome.md [hunk 1]
```diff
@@ -0,0 +1,382 @@
+---
+id: 80
+title: "Why is Collector Pattern so Awesome"
+perex: '''
+    How to achieve *open for extension* and *closed for modification* [one of sOlid principals](https://github.com/jupeter/clean-code-php#openclosed-principle-ocp)?
```
Comment (carusogabriel) [inline source/_posts/2018/2018-03-08-why-is-collector-pattern-so-awesome.md]:
```diff
-one of sOlid principals
+one of solid principals
```

File: source/_posts/2018/2018-03-08-why-is-collector-pattern-so-awesome.md [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+final class RectorCollector
+{
+    public function addRector(RectorInterface $rector): void
+    {
+        $this->retcors[] = $rector; 
+    }
+}
+```
+
+### What we want?
```
Comment (carusogabriel) [inline source/_posts/2018/2018-03-08-why-is-collector-pattern-so-awesome.md]:
```diff
-What we want?
+What do we want?
```

File: source/_posts/2018/2018-03-08-why-is-collector-pattern-so-awesome.md (hunk 1 above)
Comment (TomasVotruba) [reply source/_posts/2018/2018-03-08-why-is-collector-pattern-so-awesome.md]: That's intentional to refer to O, aka open closed

**Comment by TomasVotruba** [comment]
Thank you Gabriel!

**Pull request merged**